design/one-pager-package-format-v2.md

```diff
@@ -64,21 +57,19 @@ manager might unpack, parse, and install the aforementioned configuration. This
 document therefore proposes that two key 'views' of package metadata exist:
 
 * The 'on disk format(s)'. The format authored by a platform operator.
-* The 'output stream'. The single supported machine readable format.
+* The 'packaged format'. The single supported machine readable format.
 
-This document proposes that the output stream be a [YAML stream] emitted to
-stdout when the package's OCI image [`entrypoint`][entrypoint] is invoked. The
-'on disk format' could take many forms - for example static files or a Python
-script - as long as that format is rendered as an output stream when invoked by
-package consumers.
+This document proposes that the packaged format be a [YAML stream] stored at
+`/package.yaml` at the root of an OCI image. The 'on disk format' could take
+many forms - for example static files or a Python script - as long as that
+format is compiled to a `/package.yaml` file in an OCI image.
 
-### The Output Stream
+### The Packaged Format
 
-All software that consumes a package must do so via its output stream - a YAML
-stream emitted to stdout when the package's OCI image entrypoint is invoked. The
-output stream will consist of a `Provider` or `Configuration` YAML document
-(described below) and a series of Crossplane-specific YAML documents, which must
-be either:
+All software that consumes a package must do so via its packaged format - a YAML
+stream stored as `/package.yaml` in a single-layer OCI image. The packaged
```

Comment:
Do we have a strict requirement that it be single-layer?

Reply:
I think package.yaml being the first layer is fine requirement for the future. Images would be well suited maybe in an extra layer, but this is kind of digging into implementation and maybe best to wait until ideas like oras can land with more time in the future to be more opinionated if it makes sense.

Reply:
Yeah, I was going more for 'first layer' than 'single layer'. I was mostly intending to communicate 'you don't need to include an entire OS in your package'. Will rephrase.